Help/cobot_help.rst

```diff
@@ -0,0 +1,122 @@
+Usage of cobot
+==============
+
+.. note::
+
+Display all of the help commands that Hubot knows about
+---------------------------------------------------------
+
+cobot help
+
+::
+
+    cobot help
+
+Display all help comments that match <query>
+----------------------------------------------
+
+cobot help <query>
+
+::
+
+    cobot help explain
+
+Assigns issus <issus_link> to user
+------------------------------------
+
+conbot assign <issus_link>
+
+::
+
+    cobot assign https://github.com/coala/coala/issues/3845
+
+Unassign the user from the issus <issue_link>
+------------------------------------------------
+
+cobot unassign <issus_link>
+
+::
+
+    cobot unssign https://github.com/coala/coala/issues/3845
+
+Create a Github issus given the title and repository
+-------------------------------------------------------
+
+cobot new issus
+
+::
+
+    cobot new issus cobot "Add example of assigning an issue"
+
+Marks the given pull request as work in process/pending review
+-----------------------------------------------------------------
+
+cobot mark (wip|pending) <pull URL>
+
+::
+
+    cobot mark wip https://github.com/coala/projects/pull/237
+
+Returns the info we have on your topic
+----------------------------------------
```

Comment:
Something like "Querying for explanation on some pre-defined topics" may be better.